test/helix/impl/props_test.cljc

```diff
@@ -105,7 +105,14 @@
 
      (t/is (eq (impl/native-props {:foo "bar"
                                    & nil})
-               #js {:foo "bar"}))))
+               #js {:foo "bar"}))
+
+     (t/is (eq (impl/native-props {:foo "bar"
+                                   & nil})
+               #js {:foo "bar"}))
```

Comment:
Is this a duplicate of the test above?

Reply:
my fault, missed that while I was playing around with test setup 

cleaned up that commit below